### ISSUE TITLE
Add RPB Sachgruppen to RDF

### ISSUE THREAD

**Issue by acka47** [hbz/lobid#298]
The notations are found in 700l alongside the notations from the [RPB classification scheme](http://purl.org/lobid/rpb). Example source (snippet) to be added as soon as the MAB XML is there again.

**Comment by acka47** [comment]
Do we need specific properties for RPB Sachgruppen and classification scheme? For NWBib we have http://purl.org/lobid/lv#nwbibsubject and http://purl.org/lobid/lv#nwbibspatial.

**Comment by acka47** [comment]
Depends on https://github.com/lobid/vocabs/issues/46.

**Comment by acka47** [comment]
Example source from test file [HT018895767](http://lobid.org/hbz01/HT018895767) (snippet):

```xml
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">rpb784010</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">120</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">330</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">106</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">139</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">130</subfield>
</datafield>
<datafield tag="700" ind1="l" ind2="1">
  <subfield code="a">131</subfield>
</datafield>
```

I suggest adding a specific RPB property (`http://purl.org/lobid/lv#rpbsubject`) for listing entries from both schemes. In this case it would look like this (for API 2.0):

```json
{
   "id":"http://lobid.org/resources/HT018895767#!",
   "rpbsubject":[
      "http://purl.org/lobid/rpb#n784010",
      "https://w3id.org/lobid/rpb2#120",
      "https://w3id.org/lobid/rpb2#330",
      "https://w3id.org/lobid/rpb2#106",
      "https://w3id.org/lobid/rpb2#139",
      "https://w3id.org/lobid/rpb2#130",
      "https://w3id.org/lobid/rpb2#131"
   ]
}
```

@jschnasse @aquast Is this a good solution?

**Comment by acka47** [comment]
Another question: Would it be enough to have it in data 2.0?

**Comment by acka47** [comment]
> Would it be enough to have it in data 2.0?

I just talked to @jschnasse and he needs it in 1.0. I will start with implementing it for 2.0, though.

**Comment by acka47** [comment]
When https://github.com/hbz/lobid-resources/pull/78 we have implemented the changes for API 2.0. What's missing is lobid 1.0 where edoweb actually needs the changes. Assigning @fsteeg to implement this for API 1.0.

**Comment by fsteeg** [comment]
Deployed to production, closing. See `rpbSubject` in:

http://lobid.org/resource?format=full&id=HT018895767
